.vscode/settings.json

```diff
@@ -20,5 +20,8 @@
     ],
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true,
-    "prettier.requireConfig": true
+    "prettier.requireConfig": true,
+    "[typescript]": {
+        "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
why this?

Reply:
Whoops! That's vscode's doing, will remove